sycl/test-e2e/NonUniformGroups/tangle_group_algorithms.cpp

```diff
@@ -0,0 +1,137 @@
+// RUN: %clangxx -fsycl -fsycl-device-code-split=per_kernel -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
@steffenlarsen suggested I add this to one of the other tests.  Steffen, could you please advise on what the comment should say here?

Reply:
Since we're only testing with a single sub-group size in this test, I think it will be okay to not have the option. The reason we needed it in the other test was because it would try with different sub-group sizes, which isn't currently split correctly so the binaries would have potentially invalid kernels together with valid ones, potentially causing build failures when launching valid kernels.

Reply:
Ah, ok.  Thanks for the explanation.  Removed the flag in 92c0b10f10681384777b7c7905496cf56fc724fa.